Wikipedia/UI-V5/WMFTableOfContentsViewController.swift

```diff
@@ -129,7 +134,15 @@ public class WMFTableOfContentsViewController: UITableViewController, WMFTableOf
         super.viewDidDisappear(animated)
         deselectAllRows()
     }
-
+    
+    public override func viewWillTransitionToSize(size: CGSize, withTransitionCoordinator coordinator: UIViewControllerTransitionCoordinator) {
+        
+        coordinator.animateAlongsideTransition({ (context) -> Void in
```

Comment:
hmm this reads a bit weird in swift. I'm guessing the second closure (which is indented strangely) is the completion block? maybe we should use the labels in this instance, as the "implicit trailing closure as last parameter" feature works best for functions with only one parameter (or parameters that are more clearly labeled).